src/core_plugins/timelion/server/series_functions/legend.js

```diff
@@ -30,47 +31,86 @@ export default new Chainable('legend', {
     {
       name: 'position',
       types: ['string', 'boolean', 'null'],
-      help: 'Corner to place the legend in: nw, ne, se, or sw. You can also pass false to disable the legend',
+      help: i18n.translate('timelion.help.functions.legend.args.positionHelpText', {
+        defaultMessage:
+          'Corner to place the legend in: nw, ne, se, or sw. You can also pass false to disable the legend',
+      }),
```

Comment:
Add description: `"nw", "ne", "se", "sw" and "false" are keywords and must not be translated.`